### DISCUSSION TITLE
Delete products improvements

### DISCUSSION THREAD

**Discussion by boudo2017** [Blair2004/NexoPOS#514] [Ideas]
Hi,
you have to restrict deleting product that used for another transaction like sales or purchases. you can add possibility to make it inactive or not for sale .
orders can be refunded, some products too.
regards.

**Comment by Blair2004** [comment]
Products can be deleted and still referenced on the order. We keep a reference of the product on the order products (including the name). That option would have been handy if we could have force delete option which is not easy to implement. It's yet possible to disable a product for sale (no more action is needed there).

I can't understand "order can be refunded some product too". On NexoPOS products are refunded. If all products are refunded the order is marked as completely refunded.
